medcat/utils/meta_cat/data_utils.py

```diff
@@ -33,6 +34,8 @@ def prepare_from_json(data: Dict,
                 {'Experiencer': 'Patient'} - Take care that the CASE has to match whatever is in the data. Defaults to `{}`.
         lowercase (bool):
             Should the text be lowercased before tokenization. Defaults to True.
+        cui_filter:
```

Comment:
Why was this moved around and given less documentation? Doesn't look like there would be a reason for that.

Reply:
I've fixed this